src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -41,6 +41,59 @@ tags:
       </request>
       ```
       ... to this endpoint: `POST /request?cmd=create`
+
+
+      ## Workflow 2: Add a Review
```

Comment:
When I read this, I thought it was just referring to the `Add a Review` action we have in our UI, which simply adds a reviewer.

![Request_832772_Submit_check_email_openSUSE_Build_Service](https://user-images.githubusercontent.com/2581944/114389235-0e765300-9b95-11eb-8399-809ff26d7acb.png)

As this section explains both how to add a reviewer and how to handle the review. I suggest changing the title to something like: _"Review a Request"_ or something similar.

Of course, the action in the UI can be renamed, but that's another story.

Reply:
I changed the title to "Review a Submit Request". Please check it out again.